### PR TITLE
Updatev0.8

### PR DESCRIPTION
**Pull request by xuehui1991**
Solve conflict from v0.8 to master

### CODE REVIEW

File: docs/en_US/CommunitySharings/HpoComparision.md [hunk 1]
```diff
@@ -5,17 +5,17 @@ Comparison of Hyperparameter Optimization algorithms on several problems.
...
+- [Random Search](../BuiltinTuner.md)
+- [Grid Search](../BuiltinTuner.md)
+- [Evolution](../BuiltinTuner.md)
+- [Anneal](../BuiltinTuner.md)
+- [Metis](../BuiltinTuner.md)
+- [TPE](../BuiltinTuner.md)
+- [SMAC](../BuiltinTuner.md)
+- [HyperBand](../BuiltinTuner.md)
+- [BOHB](../BuiltinTuner.md)
+
```
Comment (leelaylay) [inline docs/en_US/CommunitySharings/HpoComparision.md]: Why to remove the `#Tuner` in these links?

Comment (xuehui1991) [reply docs/en_US/CommunitySharings/HpoComparision.md]: Please to check another PR. Cannot show #Tuner in readdocs.

**Comment by chicm-ms** [comment]
suggest not to submit PR to v0.8 for solving conflict.

File: docs/en_US/Release.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
 # ChangeLog
 
+# Release 0.8 - 6/4/2019
+## Major Features
```
Comment (scarlett2018) [inline docs/en_US/Release.md]: Add the issue links for users to reference.

**Pull request closed without merging**